subprojects/plugins/src/main/java/org/gradle/jvm/application/tasks/CreateStartScripts.java

```diff
@@ -173,6 +174,19 @@ public void setOutputDir(File outputDir) {
         this.outputDir = outputDir;
     }
 
+    /**
+     * The directory to write the scripts into in the distribution.
+     */
+    @Input
+    @Optional
```

Comment:
This shouldn't be `Optional`. A property is optional means it can be `null`. This property can't be `null`.